lib/globalize/active_record/act_macro.rb

```diff
@@ -42,7 +35,6 @@ def allow_translation_of_attributes(attr_names)
 
       def apply_globalize_options(options)
         options[:table_name] ||= "#{table_name.singularize}_translations"
-        options[:foreign_key] ||= class_name.foreign_key
```

Comment:
What do you by "we can't find it"?

We still can specify foreign_key by hand via `translates :title, foreign_key: 'page_id'`.
In case there's no `options[:foreign_key]` in `setup_translates!`, we call has_many first, to create association.

In any case, rails would create a reflection of `translations` association that would contain `foreign_key` - either set manually via `options` or derived magically by Rails.

And after that happened, it adds resulted foreign_key to translated_options anyway.